node/src/components/consensus/consensus_protocol.rs

```diff
@@ -71,6 +71,7 @@ pub(crate) enum ProtocolOutcome<I, C: Context> {
     /// TODO: Add more details that are necessary for block creation.
     CreateNewBlock {
         block_context: BlockContext,
+        past_values: Vec<C::ConsensusValue>,
```

Comment:
Is the `TODO` two lines above still valid? I am in favor of removing them if they're not.